src/templates/en/2019/featured_chapters.json

```diff
@@ -0,0 +1,183 @@
+    "stat_3": "82%",
+    "stat_label_3": "Sites that could save 1Mb by optimising caching better"
+   },
+   "cdn": {
+    "quote": "\"Use a Content Delivery Network\" was one of Steve Souders original recommendations for making web sites load faster. It's advice that remains valid today, and in this chapter of the Web Almanac we're going to explore how widely Steve's recommendation has been adopted, how sites are using Content Delivery Networks (CDNs), and some of the features they're using.",
+    "stat_1": "20%",
+    "stat_label_1": "Home pages served by CDN",
+    "stat_2": "9.61%",
+    "stat_label_2": "Homepages served by most popular CDN (Cloudflare)",
+    "stat_3": "30%",
+    "stat_label_3": "3P CDN requests that use Google"
+   },
+   "page-weight": {
+    "quote": "The common argument as to why page size doesn't matter anymore is that, thanks to high-speed internet and our souped-up devices, we can serve massive, complex (and massively complex) pages to the general population. This assumption works fine, as long as you're okay with ignoring the vast swathe of internet users who don't have access to said high-speed internet and souped-up devices.",
+    "stat_1": "10%",
+    "stat_label_1": "Sites sending more than 6 MB of Data",
+    "stat_2": "434 KB",
+    "stat_label_2": "Increase in median desktop size in last year",
+    "stat_3": "69",
+    "stat_label_3": "Median requests per home page"
+   },
+   "resource-hints": {
+    "quote": "Resource hints provide <em>hints</em> to the browser about what resources will be needed soon. The action that the browser takes as a result of receiving this hint will vary depending on the type of resource hint; different resource hints kick off different actions. When used correctly, they can improve page performance by giving a head start to important anticipated actions.",
+    "stat_1": "29%",
+    "stat_label_1": "Sites using <code>dns-prefetch</code>",
+    "stat_2": "88%",
+    "stat_label_2": "Resource hints using the <code>as</code> attribute.",
+    "stat_3": "0.04%",
+    "stat_label_3": "Usage of priority hints"
+   },
+   "http2": {
+    "quote": "HTTP/2 was the first major update to the main transport protocol of the web in nearly 20 years. It arrived with a wealth of expectations: it promised a free performance boost with no downsides. More than that, we could stop doing all the hacks and work arounds that HTTP/1.1 forced us into, due to its inefficiencies. Bundling, spriting, inlining, and even sharding domains would all become anti-patterns in an HTTP/2 world, as improved performance would be provided by default. This chapter examines how this relatively new technology has fared in the real world.",
+    "stat_1": "95%",
+    "stat_label_1": "Users who can use HTTP/2",
+    "stat_2": "27.83%",
+    "stat_label_2": "Requests with HTTP/2 prioritisation issues",
+    "stat_3": "8.38%",
+    "stat_label_3": "Sites supporting QUIC"
+   }
+  }
+  
```

Comment:
```suggestion

```

I think due to whitespace in the last line, the EOF isn't detected.

Reply:
Fixed thanks!